XenAdmin/Wizards/HAWizard.cs

```diff
@@ -159,7 +159,8 @@ protected override bool RunNextPagePrecheck(XenTabPage senderPage)
         {
             var pageType = senderPage.GetType();
 
-            if (pageType == typeof(Intro) && !_rbacNeeded || pageType == typeof(RBACWarningPage))
+            if ((pageType == typeof(Intro) && !_rbacNeeded || pageType == typeof(RBACWarningPage)) &&

```

Comment:
I don't think you need this change at all. Unless I'm missing something, my dev testing shows that simply renaming PopulatePageControls->PopulatePage does the job.

Reply:
The `IsFirstLoad` check makes sure the control changes aren't reset if the user goes back to the intro page.